app/controllers/users_controller.rb

```diff
@@ -333,6 +333,17 @@ def github_revoke
     (redirect_to user_path(id: @user.id)) && return
   end
 
+  action_auth_level :reset_password, :administrator
+  def reset_password
+    sign_out @user
+    @user = User.find(params[:id])
+    raw, enc = Devise.token_generator.generate(User, :reset_password_token)
+    @user.reset_password_token = enc
+    @user.reset_password_sent_at = Time.current
+    @user.save(validate: false)
+    redirect_to edit_password_url(@user, reset_password_token: raw)
```

Comment:
We should include some sort of flash indicating that a) password was reset for user $x and also b) email was sent, something along those lines

Reply:
Okay, added this.